README.markdown

```diff
@@ -6084,6 +6085,40 @@ This feature was first introduced in the `v0.2.1rc15` release.
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.re.opt
```

Comment:
To quote the comment at the beginning of this file:

```
<!---
Don't edit this file manually! Instead you should generate it by using:
    wiki2markdown.pl doc/HttpLuaModule.wiki
-->
```

In short, you are editing the wrong file :)